src/javascript/binary/websocket_pages/user/ico_portfolio.js

```diff
@@ -35,7 +37,7 @@ const ICOPortfolio = (() => {
         let status_text = 'Ended';
         if (/unsuccessful/i.test(long_code)) {
             status_text = 'Refund Bid';
-        } else if (/successful/i.test(long_code)) {
+        } else if (/ successful/i.test(long_code)) {
```

Comment:
will long code always be in english?

Reply:
@negarn No, but the backend doesn't provide any other information to determine the bid status.